### ISSUE TITLE
People tab not updated on PMing someone for the first time

### ISSUE THREAD

**Issue by agrawal-d** [zulip/zulip-mobile#3799]
I private messaged some person for the first time. After going back to the People screen, I expected to see their name in the list, I even browsed around some random streams for a while, and checked it again, but it appeared only after force closing and restarting the app.

![image](https://user-images.githubusercontent.com/7714968/72383756-1f41da00-3742-11ea-851b-c81443f3b4f0.png)
 ^ This did not appear until app restart.  The user/bot had to be manually searched for, to continue the conversation.
(Using the Android app version 26.20.143)

**Comment by ray-kraesig** [comment]
This is not *quite* a duplicate of #3654 (sent vs. received), but I believe it has the same underlying issue and will be resolved by the same fix.

**Comment by chrisbobbe** [comment]
@gnprice Would you say the testing you mentioned at https://github.com/zulip/zulip-mobile/issues/3460#issuecomment-584925763 was also sufficient to confirm this issue is fixed by https://github.com/zulip/zulip-mobile/pull/3877?

**Comment by chrisbobbe** [comment]
> @gnprice Would you say the testing you mentioned at #3460 (comment) was also sufficient to confirm this issue is fixed by #3877?

I just verified that that same fix (https://github.com/chrisbobbe/zulip-mobile/commit/df864186bf3ca243e1791074a90c00a4cce07fb3) also fixes this issue, so, closing.
